packages/curve-ui-kit/src/themes/components.ts

```diff
@@ -97,6 +104,15 @@ export const createComponents = (design: DesignSystem, typography: TypographyOpt
       },
     },
   },
+  MuiList: {
+    styleOverrides: {
+      root: {
+        '&:focus': {
+          outline: 'none',
```

Comment:
Are you sure we want to disable this? It might be useful for accessibility. How about `focus-visible`?
For buttons and icon buttons I use `'&:focus-visible': { borderColor: Focus_Outline }` as discussed with UX.

Reply:
The menu list has an auto-focus so you can use keyboard navigation immediately to navigate the list. However, that means it immediately renders with a damn awful black border. I could change `&:focus` to `&:focus-visible`, but showing any kind of outline is not an option, so neither would be using `Focus_Outline`

![image](https://github.com/user-attachments/assets/4981a5c6-3b34-4251-8959-2be40d564f7f)